Example/Sources/ConversationViewController.swift

```diff
@@ -82,28 +82,32 @@ internal class ConversationViewController: MessagesViewController {
     
     @objc func handleTyping() {
         
-        defer {
-            isTyping = !isTyping
-        }
-        
-        if isTyping {
-            
-            messageInputBar.topStackView.arrangedSubviews.first?.removeFromSuperview()
-            messageInputBar.topStackViewPadding = .zero
-            
-        } else {
-            
-            let label = UILabel()
-            label.text = "nathan.tannar is typing..."
-            label.font = UIFont.boldSystemFont(ofSize: 16)
-            messageInputBar.topStackView.addArrangedSubview(label)
-            messageInputBar.topStackViewPadding.top = 6
-            messageInputBar.topStackViewPadding.left = 12
-            
-            // The backgroundView doesn't include the topStackView. This is so things in the topStackView can have transparent backgrounds if you need it that way or another color all together
-            messageInputBar.backgroundColor = messageInputBar.backgroundView.backgroundColor
-            
-        }
+        isTyping = !isTyping
+        setTypingIndicatorHidden(!isTyping, animated: true, completion: { _ in
+            self.messagesCollectionView.scrollToBottom(animated: true)
+        })
+//        defer {
```

Comment:
This code is commented out, you planning on doing something with it?

Reply:
Once this gets merged I will be updating the entire example app with more detailed examples for the TypingIndicator and custom cells.